tests/queries/0_stateless/02481_array_join_with_map.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS arrays_test;
+
+CREATE TABLE arrays_test
+(
+    s String,
+    arr1 Array(UInt8),
+    arr2 Array(UInt8), 
+    map1 Map(UInt8, String),
+ map2 Map(UInt8, String)
+) ENGINE = Memory;
+
+INSERT INTO arrays_test
+VALUES ('Hello', [1,2], [1], map(1, '1', 2, '2'), map(1, '1')), ('World', [3,4,5], [3,4], map(3, '3', 4, '4', 5, '5'), map(3, '3', 4, '4')), ('Goodbye', [], [], map(), map());
+
+select s, arr1 from arrays_test  array join arr1;
+
+select s, arr1 from arrays_test  left array join arr1;
+
+select s, arr1, arr2 from arrays_test  array join arr1, arr2 settings enable_unaligned_array_join = 1;
+
+select s, arr1, arr2 from arrays_test  left array join arr1, arr2 settings enable_unaligned_array_join = 1;
```

Comment:
Is the behavior of these queries changed? It uses only arrays and should be covered by existing test cases.

Reply:
I'll remove the cases of arrays.